package.json

```diff
@@ -14,6 +14,7 @@
     "lint:css": "stylelint src/**/*.{astro,module.css,css}",
     "lint:css:fix": "stylelint --fix src/**/*.{astro,module.css,css}",
     "prettier:astro": "prettier --write src/**/*.astro",
+    "script:extract-icon-names": "node ./scripts/extractIconNames.js",
```

Comment:
When this runs, it scans ubie-icon, extracts the filename, and outputs it to json